spec/features/eq_model_spec.rb

```diff
@@ -100,5 +100,14 @@
         it_behaves_like 'cannot see pending reservations'
       end
     end
+
+    context 'with requirement' do
```

Comment:
Let's move this to a new file, say `spec/features/catalog_requirements_spec.rb`. This pattern of grouping a whole bunch of feature specs together is not great so I'd prefer to move away from that.